appveyor.yml

```diff
@@ -1,5 +1,5 @@
 version: 5.7.0.{build}
-image: Visual Studio 2017
+image: Visual Studio 2019
```

Comment:
Checking the built artifacts, the `icon` declaration is missing in packages .nuspec. I suspect it requires a VS2019 setup.